Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -202,6 +202,97 @@ - (BOOL)containsPasteboardTypes:(NSArray*)pasteboardTypes {
 
 /**
  @Status Caveat
+ @Notes only support one item due to platform limit.
+ */
+- (NSArray<NSDictionary<NSString*, id>*>*)items {
+    WADDataPackageView* wasDataPackageView = _getClipboardContent();
+    id ret = [NSMutableDictionary dictionary];
```

Comment:
i am changing its value, not sure what do you mean?

---
In reply to: [87522391](https://github.com/Microsoft/WinObjC/pull/1358#discussion_r87522391) [](ancestors = 87522391)